### DISCUSSION TITLE
Welcome to Wissenschaftliches-Arbeiten Discussions!

### DISCUSSION THREAD

**Discussion by 100gLeberwurst** [100gLeberwurst/Wissenschaftliches-Arbeiten#40] [General]
Hallo Habibis,

hier können wir unsere Ergebnisse diskutieren, Meinung austauschen und unterdrücken und unserer unendlichen Liebe zu Daten und Analysen huldigen.

Ich freue mich von euch zu hören und mich von euren geistreichen Beiträgen befruchten zu lassen.

Mit den besten Grüßen,
die Leberwurst (100g davon)

**Comment by QQTob** [comment]
Dies waren meine Beobachtungen bis jetzt:
-Mathe LK wurde gesamplet mit 66% für Ja
-Interesse Mathe ist fast gleichverteilt, aber mit 0 Ausprägungen für 1 und wenig Prozent für 2 und 7
-Interesse Programmieren ähnelt etwas einer Normalverteilung 
-Studienfächer wurden gesamplet mit Prozenten: 40,30,20,10
-Alter und Mathe LK haben keinen Zusammenhang
-Menschen die ein höheres mathematisches Interesse und geringeres Interesse an Programmieren haben, hatten oft Mathe LK
-Andersrum hatten Leute mit hoher Interesse an Programmieren und niedrigerem Mathe Interesse oft keinen Mathe LK

**Reply by 100gLeberwurst** [reply]
> -Studienfächer wurden gesamplet mit Prozenten: 40,30,20,10 

Welches Studienfach wurde denn mit welchem Prozentsatz gezogen?

**Reply by QQTob** [reply]
Statistik 40%, DataScience 30%, Informatik 20%, Mathe 10%

**Reply by 100gLeberwurst** [reply]
Vielen Dank für das schnelle Feedback!

**Comment by thorbenkiehl** [comment]
Für mich waren die Erkenntnisse der Statistiken ,  dass die Studenten, die Programmieren am meisten mögen, nie Mathe, Statistik oder Data Science studieren sondern nur Informatik.
Auch generell wird höchstens Data  Science studiert bei generellem Interesse am Programmieren

**Comment by RudiZul** [comment]
Dann mal hier die Auflösung:

- Eine Verteilung der Studienfächer von Statistik 40%, DataScience 30%, Informatik 20%, Mathe 10% ist nicht ganz richtig. Statistik und Data Science lagen eigentlich beide bei 35%. Die anderen beiden Werte stimmen allerdings.

- Ein hohes Interesse an Programmieren ist bei Studierenden der Fächer Statistik und Mathe am seltensten (hier war ein Maximum von 3 bzw. 4 möglich)

- Andersherum war ein hohes Interesse an Mathematik bei Informatik-Studierenden am geringsten (die Ausprägungen 2-6 waren verteilt mit Wahrscheinlichkeiten 0.1,0.3,0.3,0.2,0.1)

- Die Ausprägung Mathe-LK wurde auch abhängig vom Studienfach gewählt und zwar wie folgt ("Ja"-"Nein"): Statistik: 0,7-0,3 | DataScience: 0,7-0,3 | Mathe: 0,9-0,1 | Informatik 0,6-0,4

- Auch richtig erkannt wurde, dass kein Zshg. zwischen dem Alter und dem Mathe-LK besteht. Dies war (wie schon geschrieben abhängig vom Studienfach)

Falls ihr noch offene Fragen habt, könnt ihr ja hier nochmal schreiben
